stacks_queues/stack.py

```diff
@@ -21,18 +21,22 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.empty():
+            StackEmptyException("The stack is empty!")
+            return None
+        else:
+            return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return True if self.store.empty() else False
```

Comment:
👀 If you look at the LinkedList `empty` implementation, it already returns the boolean for you! 

```suggestion
        return self.store.empty()
```